docs/layouts/shortcodes/cli-installation-requirements.md

```diff
@@ -0,0 +1,9 @@
+Before you install it, there are some requirements that must be met first.
+
+You need to have:
+
+* [PHP 5.5.9+](https://www.php.net/manual/en/install.php) with the following extensions installed: `curl`, `json`, `mbstring`, `pcre`, and `phar`.
+* [Git](https://docs.github.com/en/get-started/quickstart/set-up-git) is the open source version control system used to manage your project. Any change you make to your codebase needs to be committed via Git. You can see all the Git commit messages of an environment in the `Environment Activity` feed of the [management console](/administration/web/_index.md) for each project you create.
```

Comment:
:hammer_and_wrench: I guess I overlooked this (probably because it wasn't really new with this PR), but I would suggest linking to a page that described the process for downloading and installing Git. The linked GitHub page doesn't do that, it links to: https://git-scm.com/downloads So that should probably be what we link to.